### ISSUE TITLE
Smaller Image / Alpine

### ISSUE THREAD

**Issue by morriswinkler-simple** [elkarbackup/elkarbackup-docker#20]
I think it would make more sense to have an alpine image, possible with fpm or nginx. 

I think also that the installation through a Debian package is too distro specific, maybe you could pull a tar.gz with a REV from your GitHub repro and unpack it. 

Have a look at the images of Nextcloud, it is a pretty good starting point for containerizing web applications that rely on web servers like Apache or Nginx and PHP.

Also things like reinstalling the dpkg every time a container starts or pulling an install script from GitHub is not a good solution, at least don't pull it from master, use an SHA REV to make sure the file content is what you expect it to be.

**Comment by xezpeleta** [comment]
Thanks for your advice @morriswinkler-simple 

We'll keep it in mind

**Comment by xezpeleta** [comment]
Hi @morriswinkler-simple 

I'm preparing a new docker image following your advices. Take a look to the PR #23, I'd really appreciate your opinion!

Thanks

**Comment by xezpeleta** [comment]
Finally I've released a new docker image based in your advices:

https://hub.docker.com/r/elkarbackup/elkarbackup/

- Not using the deb package anymore
- Download the upstream tgz from the Github release page with a REV
- Use a SHA REV to make sure that the downloaded content is what we want

The default image is now based on _PHP:7.1-apache_ image: `elkarbackup/elkarbackup:1.3.1-apache`
- No, we don't have Alpine image: I've found some issues with it (permissions, filesystem etc). So we're now focusing in the default one with Apache. Alpine, PHP-FPM.. are in our roadmap.
- However, as it's built using multistage images, anyone can copy the web app from the `elkarbackup/elkarbackup:1.3.1` image and use it in its container:

```
COPY --from=elkarbackup/elkarbackup:1.3.1-build /app /app
```
